### PR TITLE
2021 January updates

### PR DESCRIPTION
**Pull request by rudeboybert**

### CODE REVIEW

**Comment by rudeboybert** [comment]
Hey @ismayc, please hold off on merging into master after your review. I'm still thinking about what to do about deployment

File: 92-appendixB.Rmd [hunk 1]
```diff
@@ -524,7 +524,7 @@ Note that the 95 percent confidence interval given above matches well with the o
 
 #### State conclusion {-}
 
-We, therefore, do not have sufficient evidence to reject the null hypothesis.  Our initial guess that our observed sample proportion was not statistically greater than the hypothesized proportion has not been invalidated.  Based on this sample, we have do not evidence that the proportion of all customers of the large electric utility satisfied with service they receive is different from 0.80, at the 5% level.
+We, therefore, do not have sufficient evidence to reject the null hypothesis.  Our initial guess that our observed sample proportion was not statistically different the hypothesized proportion has not been invalidated.  Based on this sample, we have do not evidence that the proportion of all customers of the large electric utility satisfied with service they receive is different from 0.80, at the 5% level.
```
Comment (ismayc) [inline 92-appendixB.Rmd:527]:
```suggestion
We, therefore, do not have sufficient evidence to reject the null hypothesis.  Our initial guess that our observed sample proportion was not statistically different from the hypothesized proportion has not been invalidated.  Based on this sample, we have do not evidence that the proportion of all customers of the large electric utility satisfied with service they receive is different from 0.80, at the 5% level.
```

File: 92-appendixB.Rmd [hunk 2]
```diff
@@ -1279,7 +1276,7 @@ Remember that in order to use the shortcut (formula-based, theoretical) approach
...
+The test statistic is a random variable based on the sample data.  Here, we want to look at a way to estimate the population mean difference $\mu_{diff}$.  A good guess is the sample mean difference $\bar{X}_{diff}$.  Recall that this sample mean is actually a random variable that will vary as different samples are (theoretically, would be) collected.  We are looking to see how likely is it for us to have observed a sample mean of $\bar{x}_{diff, obs} = -0.0804$ or smaller assuming that the population mean difference is 0 (assuming the null hypothesis is true).  If the conditions are met and assuming $H_0$ is true, we can "standardize" this original test statistic of $\bar{X}_{diff}$ into a $T$ statistic that follows a $t$ distribution with degrees of freedom equal to $df = n - 1$:
```
Comment (ismayc) [inline 92-appendixB.Rmd:1279]: Probably best to not hard code here, right?

File: 94-appendixD.Rmd [hunk 3]
```diff
@@ -500,7 +500,7 @@ with?
 
 **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What can we say about the distribution of `gain`?  Describe it in a few sentences using the plot and the `gain_summary` data frame values.
 
-**Solution**: Most of the time the gain is a little under zero, most of the time the gain is between -50 and 50 minutes. There are some extreme cases however!
+**Solution**: Most of the time the gain is a little above zero (the median is 7, meaning gain is above 0 at least 50% of the time), most of the time the gain is between -50 and 50 minutes. There are some extreme cases however!
```
Comment (ismayc) [inline 94-appendixD.Rmd:503]: This sentence might need some cleaning up since "most of the time" is repeated twice with two different situations.

**Review by ismayc — CHANGES_REQUESTED** [review]
Just a few comments here. Otherwise it looks good!

**Pull request merged**